src/containers/FileExplorer/FilePreview.tsx

```diff
@@ -152,20 +152,29 @@ class MapModelToAssetForm extends React.Component<Props, State> {
     try {
       trackUsage('FilePreview.DetectAsset', { fileId });
       const response = await sdk.post(
-        `/api/playground/projects/${sdk.project}/context/string-matcher/extract`,
+        `/api/playground/projects/${sdk.project}/context/entity_extraction/extract`,
         {
           data: {
-            pdf_file_id: fileId,
+            fileId,
```

Comment:
this looks kind of wrong extension-wise. What if you want to add more keys? :) acceptable for this, though.

Reply:
It translates to `fileId: fileId`, but is javascript syntax if key in dict has same name as variable.